e2e/integration_tests/data_flow_test.go

```diff
@@ -242,6 +245,123 @@ func Test_EndpointLinksAreAvailable(t *testing.T) {
 	}
 }
 
+func Test_GetCHPLProducts(t *testing.T) {
+	var err error
+	healthit_prod_row := store.DB.QueryRow("SELECT COUNT(*) FROM healthit_products;")
+	expected_hitp_count := 7829
+	var hitp_count int
+	err = healthit_prod_row.Scan(&hitp_count)
+	failOnError(err)
+	if hitp_count != 0 {
+		t.Fatalf("Healthit product database should initially be empty")
+	}
+
+	ctx := context.Background()
+	client := &http.Client{
+		Timeout: time.Second * 35,
+	}
+	err = chplquerier.GetCHPLProducts(ctx, store, client)
+	failOnError(err)
+
+	healthit_prod_row = store.DB.QueryRow("SELECT COUNT(*) FROM healthit_products;")
+	err = healthit_prod_row.Scan(&hitp_count)
+	failOnError(err)
+	if hitp_count != expected_hitp_count {
```

Comment:
i got a failure here. i'm guessing chpl added in products since you wrote this.
i'd modify to `hitp_count < expected_hitp_count` and say that it's at least the number that you previously retrieved.